website/docs/r/nsxt_global_default_segment_profile_template.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_global_default_segment_profile_template"
+sidebar_current: "docs-vcd-resource-nsxt-segment-profile-template"
+description: |-
+  Provides a resource to manage Global Default NSX-T Segment Profile Templates.
+---
+
+# vcd\_nsxt\_global\_default\_segment\_profile\_template
+
+Provides a resource to manage Global Default NSX-T Segment Profile Templates.
+
+Supported in provider *v3.11+* and VCD 10.4.0+ with NSX-T. Requires System Administrator privileges.
+
+## Example Usage
+
+```hcl
+resource "vcd_nsxt_global_default_segment_profile_template" "singleton" {
+  vdc_networks_default_segment_profile_template_id  = vcd_nsxt_segment_profile_template.complete.id
+  vapp_networks_default_segment_profile_template_id = vcd_nsxt_segment_profile_template.empty.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `vdc_networks_default_segment_profile_template_id` - (Optional) - Global Default Segment Profile
+  Template ID for all VDC Networks
+* `vapp_networks_default_segment_profile_template_id` - (Optional) - Global Default Segment Profile
+  Template ID for all vApp Networks
```

Comment:
BTW:

```suggestion
* `vdc_networks_default_segment_profile_template_id` - (Optional) Global Default Segment Profile
  Template ID for all VDC Networks
* `vapp_networks_default_segment_profile_template_id` - (Optional) Global Default Segment Profile
  Template ID for all vApp Networks
```

Reply:
> Both arguments are optional. What happens if we skip both of the

Well done - this has caught a bug (I had to handle the structure a bit differently). In general - it will simply unset both values if they were present (it is equivalent to `destroy`). I added a test step for it in `TestAccVcdNsxtSegmentProfileTemplate`.


> BTW:

Fixed.